From e40a9b65ae95ed6face7ce1336905f0ab0a4992b Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Tue, 05 Feb 2013 21:37:40 +0000
Subject: gzip: fix bug where you say "n" and gzip acts as if you said "y"

Problem reported for GCC 4.7 x86-64 -O2 by Allan McRae in
<http://lists.gnu.org/archive/html/bug-gzip/2013-02/msg00000.html>.
* NEWS: Document this.  Use consistent format in earlier note.
* gzip.c: Include yesno.h.
* gzip.h (yesno): Remove decl; that's yesno.h's job.
---
diff --git a/gzip.c b/gzip.c
index b9b4d64..66529e0 100644
--- a/gzip.c
+++ b/gzip.c
@@ -74,6 +74,7 @@ static char const *const license_msg[] = {
 #include "ignore-value.h"
 #include "stat-time.h"
 #include "version.h"
+#include "yesno.h"
 
                 /* configuration */
 
diff --git a/gzip.h b/gzip.h
index bf253d0..648073e 100644
--- a/gzip.h
+++ b/gzip.h
@@ -321,6 +321,3 @@ extern void fprint_off    (FILE *, off_t, int);
 
         /* in inflate.c */
 extern int inflate (void);
-
-        /* in yesno.c */
-extern int yesno (void);
--
cgit v0.9.0.2
